app/services/void_participant_declaration.rb

```diff
@@ -24,6 +24,8 @@ def make_awaiting_clawback
     if clawback.errors.any?
       raise Api::Errors::InvalidTransitionError, clawback.errors.full_messages.join(", ")
     end
+
+    check_mentor_completion
```

Comment:
should we move this to the clawback declaration service to ensure it's within that transaction?